src/index.js

```diff
@@ -746,6 +818,9 @@ export const fuzzExpression = (f = new FuzzerState, {allowIdentifierExpression =
   if (f.allowYieldExpr) {
     fuzzers = fuzzers.concat(yieldExprFuzzers);
```

Comment:
This needs brackets too, right?

Reply:
No, [it's already an array](https://github.com/shapesecurity/shift-fuzzer-js/blob/5a125892e2325e1f2b57ed41ebfa3cd6932a56f6/src/index.js#L718) (hence the plural).